tests/ui/lint/dead-code/unused-struct-derive-default.stderr

```diff
@@ -10,14 +10,19 @@ note: the lint level is defined here
 LL | #![deny(dead_code)]
    |         ^^^^^^^^^
 
-error: variant `B` is never constructed
-  --> $DIR/unused-struct-derive-default.rs:13:5
+error: variants `A` and `B` are never constructed
+  --> $DIR/unused-struct-derive-default.rs:12:5
    |
 LL | enum E {
-   |      - variant in this enum
-...
+   |      - variants in this enum
+LL |     #[default]
+LL |     A,
+   |     ^
+LL |
 LL |     B,
    |     ^
+   |
+   = note: `E` has a derived impl for the trait `Default`, but this is intentionally ignored during dead code analysis
```

Comment:
Need to verify why this arguably regresses.

Reply:
I don't remember why I marked the trait with `#[rustc_trivial_field_reads]`, but that was the reason.